pkg/vm/vm_core.go

```diff
@@ -1,18 +1,32 @@
 package vm
 
 import (
-	"errors"
 	"fmt"
 
 	"github.com/lambdaclass/cairo-vm.go/pkg/builtins"
 	"github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
 	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+	"github.com/pkg/errors"
 )
 
 type VirtualMachineError struct {
 	Msg string
 }
 
+var ErrWrongInstructionEncoding = errors.New("Wrong instruction encoding")
+var ErrFailedToFetchInstruction = errors.New("Failed to fetch instruction in address")
+var ErrNoRelocationFoundSegment = errors.New("No relocation found for execution segment")
+var ErrTraceNotRelocated = errors.New("Trace not relocated")
+var ErrUnconstraintRes = errors.New("UnconstrainedRes")
```

Comment:
```suggestion
var ErrUnconstraintRes = errors.New("UnconstrainedRes")
```
This is not descriptive enough